### PR TITLE
Update Comet v7.0.0-beta.4

### PR DESCRIPTION
**Pull request by SebiVPS**

### CODE REVIEW

File: site/next.config.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ const nextConfig = {
                                 frame-ancestors ${process.env.ADMIN_URL};
                                 upgrade-insecure-requests; 
                                 block-all-mixed-content;
+                                frame-src youtube.com https://www.youtube.com https://www.youtube-nocookie.com/;
```
Comment (SebiVPS) [inline site/next.config.js:64]: @johnnyomair without this, the YouTube video does not play

Comment (johnnyomair) [reply site/next.config.js:64]: @max-debug022 this should be okay, right?

Comment (max-debug022) [reply site/next.config.js:64]: Yes. Could be simplified to `frame-src 'self' https://*.youtube.com https://www.youtube-nocookie.com/;` though.

Comment (johnnyomair) [reply site/next.config.js:64]:
```suggestion
                                frame-src 'self' https://*.youtube.com https://*.youtube-nocookie.com;
```

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**